presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2869,6 +2869,22 @@ public void testCreateAndInsert()
         assertEqualsIgnoreOrder(actualAfterTransaction, expected);
     }
 
+    @Test
+    public void testRenameView()
+    {
+        assertUpdate("CREATE VIEW rename_view_original AS SELECT COUNT(*) as count FROM orders");
+
+        assertQuery("SELECT * FROM rename_view_original", "SELECT COUNT(*) FROM orders");
+
+        assertUpdate("ALTER VIEW rename_view_original RENAME TO rename_view_new");
```

Comment:
please add  a  test renaming view between schemas.

please add  a  test renaming view between catalogs.

Reply:
You can add a `testRenameView` for cross catalog in `AbstractTestDistributedQueries` (we can't do others until all connectors with views support rename view)

Reply:
I change the test to be cross schema.